pymobiledevice3/services/web_protocol/inspector_session.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+import json
+import logging
+
+import pymobiledevice3.exceptions
+from pymobiledevice3.cli.cli_common import print_json
+from pymobiledevice3.services.web_protocol.session_protocol import SessionProtocol
+
+logger = logging.getLogger(__name__)
+
+
+class InspectorSession:
+
+    def __init__(self, protocol: SessionProtocol, target_id: str):
+        """
+        :param pymobiledevice3.services.web_protocol.session_protocol.SessionProtocol protocol: Session protocol.
+        """
+        self.protocol = protocol
+        self.target_id = target_id
+
+    @classmethod
+    async def create(cls, protocol: SessionProtocol):
+        """
+        :param pymobiledevice3.services.web_protocol.session_protocol.SessionProtocol protocol: Session protocol.
+        """
+        await protocol.inspector.setup_inspector_socket(protocol.id_, protocol.app.id_, protocol.page.id_)
+        while not protocol.inspector.wir_events:
+            await asyncio.sleep(0)
+        created = protocol.inspector.wir_events.pop(0)
+        while 'targetInfo' not in created['params']:
+            created = protocol.inspector.wir_events.pop(0)
+        target_id = created['params']['targetInfo']['targetId']
+        logger.info(f'Created: {target_id}')
+        target = cls(protocol, target_id)
+        return target
+
+    async def js_loop(self):
```

Comment:
the js loop should be in the CLI, using `inspector_session.runtime_enable()` and `inspector_session.runtime_evaluate()`

Reply:
done